kiota_http/middleware/parameters_name_decoding_handler.py

```diff
@@ -67,30 +69,33 @@ def _get_current_options(self, request: httpx.Request) -> ParametersNameDecoding
         Returns:
             ParametersNameDecodingHandlerOption: The options to used.
         """
-        if options := getattr(request, 'options', None):
+        if options := getattr(request, "options", None):
             current_options = options.get(  # type:ignore
                 ParametersNameDecodingHandlerOption.get_key(), self.options
             )
             return current_options
         return self.options
 
-    @staticmethod
```

Comment:
any specific reason why the static method annotation was removed?

Reply:
Looking at the history (pre the 0.6.1 change) it was only marked static so it could be reused from the telemetry context. Since we aren't using it there it can go back to being a class method

Reply:
thanks for the additional information here CC @samwelkanda who has more context here

Reply:
@baywet I made it static to match the other implementations. We're re-using this method for `startTracingSpan` in the request adapter.

Reply:
I don't believe we need to use that method in the request adapter anymore, thus not needing the static modifier. I believe the current change is fine.